src/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
From https://github.com/facebook/react/blob/master/LICENSE this logo seems to be MIT licensed, which is compatible with GPL3 but we would need to track this somewhere, if we start mixing licenses inside a single codebase.

I would suggest we either remove it, or replace it with something we own, that can be covered by GPL 3 as everything else in this repo.